spec/directions/bake_chapter_summary_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'spec_helper'
+
+RSpec.describe Kitchen::Directions::BakeChapterSummary do
+
+  let(:chapter) do
+    chapter_element(
+      <<~HTML
+        <div data-type='page' id ="01">
```

Comment:
Add an introduction page to test that `chapter.pages('$:not(.introduction)')` is not iterating over it.